app/components/createService/createservice.html

```diff
@@ -435,9 +433,70 @@
                     Operation order on failure.
                   </p>
                 </div>
-
               </div>
               <!-- !order-input -->
+              <!-- restartCondition-input -->
```

Comment:
Could you add a "Restart policy" section before this input as it is done in the Resources & Placement section?

Reply:
Of course! I'll also move the html code to its own file like resources-placement.html

Reply:
This will be how it looks as suggested.

![restart_policy_sections](https://user-images.githubusercontent.com/30386061/33727557-af732732-db78-11e7-8c2d-6baa7f2e31dc.png)